src/Microsoft.AspNetCore.Blazor.Razor.Extensions/ComponentLoweringPass.cs

```diff
@@ -154,31 +131,5 @@ private void RewriteUsage(TagHelperIntermediateNode node, TagHelperDescriptor ta
                 }
             }
         }
-
-        private static bool HasComplexChildContent(IntermediateNode node)
-        {
-            if (node.Children.Count == 1 &&
-                node.Children[0] is HtmlAttributeIntermediateNode htmlNode &&
-                htmlNode.Children.Count > 1)
-            {
-                // This case can be hit for a 'string' attribute
-                return true;
-            }
-            else if (node.Children.Count == 1 &&
-                node.Children[0] is CSharpExpressionIntermediateNode cSharpNode &&
-                cSharpNode.Children.Count > 1)
-            {
-                // This case can be hit when the attribute has an explicit @ inside, which
-                // 'escapes' any special sugar we provide for codegen.
-                return true;
-            }
-            else if (node.Children.Count > 1)
-            {
-                // This is the common case for 'mixed' content
-                return true;
-            }
-
-            return false;
-        }
```

Comment:
All of the removals here are dead code. I wrote another pass earlier that sanitizes things, and I forgot to remove it from here.